src/main/scala-sbt-0.13/interplay/PlaySbtCompat.scala

```diff
@@ -0,0 +1,121 @@
+package interplay
+
+import java.io.File
+
+import sbt._
+import sbt.compiler.{AnalyzingCompiler, Eval, JavaTool}
+
+/**
+ * An object that offers compatibility helpers when creating builds that must work in
+ * different versions of sbt. This version is designed to be compatible with sbt 1.0.
```

Comment:
`This version is designed to be compatible with sbt 0.13.`

`0.13` instead of  `1.0`.

Reply:
:+1: